project/Dependencies.scala

```diff
@@ -1,5 +0,0 @@
-import sbt._
-
-object Dependencies {
-  lazy val scalaTest = "org.scalatest" %% "scalatest" % "3.2.11"
-}
```

Comment:
While this is fine, you'll see the usage of a dependency file very often in scala code bases

Reply:
Good to know, I will keep it in mind!